sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Vendoring/Shared/Shims/NullableAttributes.cs

```diff
@@ -20,19 +20,5 @@ internal sealed class AllowNullAttribute : Attribute
     internal sealed class MaybeNullAttribute : Attribute
     {
     }
-
-    /// <summary>Specifies that when a method returns <see cref="ReturnValue"/>, the parameter will not be null even if the corresponding type allows it.</summary>
```

Comment:
Why this was removed from vendored code?

Reply:
This method must be removed as it conflicts with the NullableAttribute class that we have in our project.

Reply:
I don't think so. Vendoring is always going to require some minor changes. This file in particular is in the Shared directory and we already need to do extra work to merge these files between -dotnet and -contrib repos.

Our copy of this file actually lives in the Exporter repo and is shared with all our projects. As the longterm goal is to remove the vendored code, I would prioritize our copy.